usagov/vote-gov-drupal
# feature/VOTE-809-header-display

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-809

## Description (optional)
Add components to the header as specified by the Front-end build document. The scope of this ticket is to display the data, not to apply any styling.

### Testing steps
1. Check out the feature/VOTE-809-header-display branch
2. Run lando retune
3. While logged in as admin, create two or more blocks of type "site-wide alert"
4. Log out admin and view the header as a site visitor
5. Verify all four components have data displayed within the header region
